frodo.js

```diff
@@ -4,83 +4,90 @@
  * This plugin performs initialization (like calling mermaid.initialize())
```

Comment:
SCRIPT-The way you've refactored this complex method into simpler ones is great. It makes the code much more maintainable.